Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader

```diff
@@ -473,17 +475,13 @@ Shader "SR_Triangles" {
                 bool Root = Next_Q25;
 
                 int vxix = 0;
-                int strip = 0;
-                while (strip < stripCount) {
-                    int i = 0;
-                    while (i < stripVxCount[strip]) {
-                        UNITY_TRANSFER_VERTEX_OUTPUT_STEREO(vxIn[0],vxOut[vxix]);
-                        triStream.Append(vxOut[vxix]);
-                        i += 1; vxix += 1;
-                    }
-                    triStream.RestartStrip();
-                    strip += 1;
+                int i = 0;
+                while (i < stripVxCount) {
```

Comment:
Is `stripVxCount` guaranteed to be `>1`?

Reply:
it's pretty much gauranteed to be 3

Reply:
I'm more leaning on @Railboy to sanity check this - I'm purely looking at this code from a code perspective.

Reply:
I'm out of my depth with the shader itself but from a code perspective this makes sense to me.